colcon_sanitizer_reports/sanitizer_log_parser.py

```diff
@@ -34,7 +34,16 @@
 _FIND_SECTION_END_LINE_REGEX = re.compile(r'^(?P<prefix>.*)(SUMMARY: .*Sanitizer: .*)$')
 
 
-class SanitizerLogParserOutputPrimaryKey(NamedTuple):
+SanitizerLogParserOutputPrimaryKey = NamedTuple(
+    'SanitizerLogParserOutputPrimaryKey',
+    [
+        ('package', str),
+        ('error_name', str),
+        ('stack_trace_key', str)
+    ]
+)
+
+SanitizerLogParserOutputPrimaryKey.__doc__ = (
```

Comment:
Any specific reason for choosing `__doc__` attribute for this variable, compared to regular comments that are used otherwise?

Reply:
It's a magic variable attached to types. For example
```
class Foo:
    """This will be stored at Foo.__doc__"""
    pass
```
Since I can no longer declare the SanitizerLogParserOutputPrimaryKey as a class, this is the equivalent way to attach the documentation.